### ISSUE TITLE
Production build don't work

### ISSUE THREAD

**Issue by lucavix** [AckerApple/stripe-angular#9]
Projects build with prod flag "ng build --prod" have the following error:
```javascript
Uncaught TypeError: Object prototype may only be an Object or null: undefined
    at setPrototypeOf (<anonymous>)  
    at __extends (VM43 main.c7b3fc0bbca7aa56c2db.js:109374)  
    at VM43 main.c7b3fc0bbca7aa56c2db.js:108972  
    at Object../node_modules/stripe-angular/components/StripeBank.component.js (VM43 main.c7b3fc0bbca7aa56c2db.js:109019)  
    at __webpack_require__ (VM41 runtime.618bcded6741c4c27fdf.js:79)  
    at Object../node_modules/stripe-angular/index.js (VM43 main.c7b3fc0bbca7aa56c2db.js:109284)  
    at __webpack_require__ (VM41 runtime.618bcded6741c4c27fdf.js:79)  
    at Object../node_modules/stripe-angular/StripeScriptTag.js (VM43 main.c7b3fc0bbca7aa56c2db.js:108891)  
    at __webpack_require__ (VM41 runtime.618bcded6741c4c27fdf.js:79)  
    at Object../node_modules/stripe-angular/components/StripeSource.component.js (VM43 main.c7b3fc0bbca7aa56c2db.js:109160)
```
The error is not present in development build.
Removing StripeBank from the exports resolve the problem.

my production configuration is:
```javascript
          "configurations": {
            "production": {
              "fileReplacements": [
                {
                  "replace": "src/environments/environment.ts",
                  "with": "src/environments/environment.prod.ts"
                }
              ],
              "optimization": false,
              "outputHashing": "none",
              "sourceMap": false,
              "extractCss": true,
              "namedChunks": false,
              "aot": true,
              "extractLicenses": true,
              "vendorChunk": false,
              "buildOptimizer": true,
              "budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "2mb",
                  "maximumError": "5mb"
                }
              ]
            }
```

ng ver. 7.3.8
npm ver 6.4.1
node ver v10.15.3

**Comment by AckerApple** [comment]
Have you been using this package in the past and perhaps updated to a new Angular/TypeScript and now you are seeing this issue?

This package gets enough traffic to know production building works.

My last production build was:
ng 7.2.0
npm 6.9.0
node 12.2.0

Thoughts? It's going to be some time before I can update just to "prove" it works. I'm assuming it's an update or some sort of single issue you are having.

**Comment by lucavix** [comment]
This is my first use of the package.
I try to give you a clean example.

It's not blocking for me... I fork the package and disable stripe-bank... but I can't understand why! StripeCard and StripeBank are very similar... why one works and not the other...

I keep you informed when I find the time to give you a clean project example.

**Comment by AckerApple** [comment]
Thank you kindly. I will eventually update and build the project that uses this in the not too far future.

Report when you can, I will do the same

**Comment by AckerApple** [comment]
#11 looks good by eye, I understand what it addresses. Blindly merged from iPad. Will publish to NPM after increasing version number and testing.

**Comment by AckerApple** [comment]
released to npm stripe-angular@0.5.0

Please test and close

**Comment by SudharakaP** [comment]
@AckerApple : It works. I have tested this on our web-page itself. 👍
